sqlglot/dialects/clickhouse.py

```diff
@@ -260,6 +260,7 @@ class Tokenizer(tokens.Tokenizer):
             "LOWCARDINALITY": TokenType.LOWCARDINALITY,
             "MAP": TokenType.MAP,
             "NESTED": TokenType.NESTED,
+            "NOTHING": TokenType.NULL,
```

Comment:
Let's map this to a (new) `TokenType.NOTHING` token instead of overloading `NULL`.

Reply:
No need for that, tokens are dynamically loaded in the Rust tokenizer, so you should only modify Python code.

Reply:
Ah that is quite nice. Thanks

Reply:
Updated here: cfc9329170c6b3a51376af951f1c8e1b5c9c2e88